keel-sql/src/main/resources/db/changelog/202010303-baked-images.yml

```diff
@@ -0,0 +1,46 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-baked-images
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: baked_images
+            columns:
+              - column:
+                  name: package_version
+                  type: varchar(150)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: cloud_provider
+                  type: varchar(50)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: base_os
+                  type: varchar(50)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: base_label
+                  type: varchar(50)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
Seems like we could generate these columns based on the `json` column since all the values in the insert are just derived from that object

Reply:
Done!